workflows/epigenetics/chipseq-pe-with-replicates-controls/chipseq-pe-with-replicates-controls.ga

```diff
@@ -0,0 +1,3692 @@
+            "workflow_outputs": []
+        },
+        "72": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_compute_matrix/deeptools_compute_matrix/3.5.4+galaxy0",
+            "errors": null,
+            "id": 72,
+            "input_connections": {
+                "multibigwig_conditional|bigwigfiles": {
+                    "id": 52,
+                    "output_name": "outFileName"
+                },
+                "regionsFiles_0|regionsFile": {
+                    "id": 68,
+                    "output_name": "output"
+                }
+            },
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool computeMatrix",
+                    "name": "advancedOpt"
+                },
+                {
+                    "description": "runtime parameter for tool computeMatrix",
+                    "name": "multibigwig_conditional"
+                }
+            ],
+            "label": null,
+            "name": "computeMatrix",
+            "outputs": [
+                {
+                    "name": "outFileName",
+                    "type": "deeptools_compute_matrix_archive"
+                }
+            ],
+            "position": {
+                "left": 5551.394104212098,
+                "top": 1633.9066101050357
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_compute_matrix/deeptools_compute_matrix/3.5.4+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "a60c359ec43c",
+                "name": "deeptools_compute_matrix",
+                "owner": "bgruening",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"advancedOpt\": {\"showAdvancedOpt\": \"yes\", \"__current_case__\": 1, \"binSize\": \"50\", \"sortRegions\": \"keep\", \"sortUsing\": \"mean\", \"averageTypeBins\": \"mean\", \"missingDataAsZero\": true, \"skipZeros\": true, \"minThreshold\": null, \"maxThreshold\": null, \"scale\": null, \"metagene\": false, \"transcriptID\": \"transcript\", \"exonID\": \"exon\", \"transcript_id_designator\": \"transcript_id\", \"blackListFileName\": {\"__class__\": \"RuntimeValue\"}}, \"custom_sample_labels_conditional\": {\"custom_labels_select\": \"No\", \"__current_case__\": 0}, \"mode\": {\"mode_select\": \"reference-point\", \"__current_case__\": 1, \"referencePoint\": \"center\", \"nanAfterEnd\": false, \"beforeRegionStartLength\": \"3000\", \"afterRegionStartLength\": \"3000\"}, \"multibigwig_conditional\": {\"orderMatters\": \"No\", \"__current_case__\": 0, \"bigwigfiles\": {\"__class__\": \"ConnectedValue\"}}, \"output\": {\"showOutputSettings\": \"no\", \"__current_case__\": 0}, \"regionsFiles\": [{\"__index__\": 0, \"regionsFile\": {\"__class__\": \"ConnectedValue\"}}], \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
So the beforeRegionStartLength and the afterRegionStartLength here should not be hard-coded.
Initially I thought I could come up with a way to estimate them from the MACS2 peak region widths, but maybe we should just turn them into a WF param for now.

Reply:
Should we start with the hardcoded region length for tomorrow ?

Reply:
yes, please :+1: